hack/run-e2e.sh

```diff
@@ -35,4 +35,4 @@ function cleanup {
 trap cleanup EXIT
 
 # Run e2e test
-go test ./test/e2e -v -timeout 30m
+go test ./test/e2e/kube-batch -v -timeout 30m
```

Comment:
`run-e2e.sh` script file is not used in the new Travis workflow, please update the `run-e2e-kind` file instead, also, please append test folder, not replace the older ones. For instance.
```
go test ./test/e2e/ -v -timeout 30m
go test ./test/e2e/kube-batch -v -timeout 30m
```

Reply:
Thanks @shivramsrivastava

Reply:
I have restored the e2e script.
BTW the script  ```'run-e2e-kind.sh'``` runs all the test cases under ```test/e2e``` which includes kube-batch e2e test cases too.

Thanks @TommyLike

Reply:
No, it's not running